tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAsset.scala

```diff
@@ -28,8 +30,10 @@ object PublicURI {
     .toEither
 
   def from(assetId: PublicAssetId, account: AccountId, publicUriPrefix: URI): PublicURI = {
-    val relativeURI = new URI(s"publicAsset/${account.id.value}/${assetId.asString()}")
-    PublicURI(publicUriPrefix.resolve(relativeURI))
+    val uri: URI = new URIBuilder(publicUriPrefix)
+      .setPathSegments(ImmutableList.of("publicAsset", account.id.value, assetId.asString()))
```

Comment:
What if the public uri prefix looks like `https://whatever.com/jmap` ?

As it stands this code do not preserve the path of the prefix...